packages/@okta/vuepress-site/docs/reference/releases-at-okta/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Release lifecycle
 ---
-
```

Comment:
You might want to add a link to the [Redocly page](https://developer.okta.com/docs/api/openapi/okta-management/guides/release-lifecycle/) since we're not deleting this page. If someone bookmarks this page, it would be nice to have a link instead of blank page.